llama_cpp/src/Utility/DataGenerator.h

```diff
@@ -25,20 +25,16 @@
 
 namespace llama {
 
-class RandomData {
+class DataGenerator {
  public:
-    RandomData() {}
+    DataGenerator() {}
 
     template<typename T>
-    void generateRandomData(std::vector<T> *targetVector, int count);
-};
-
-class ContinuousData {
- public:
-    ContinuousData() {}
-
+    void generateRandomData(std::vector<T> *targetVector, int count = 10);
+    template<typename T>
+    void generateContinuousData(std::vector<T> *targetVector, int count = 10, bool reverse = false);
```

Comment:
IMO, since we don't hold any data in this class, perhaps it would be nicer if all the methods are class method.